### DISCUSSION TITLE
Beta testers mailing list

### DISCUSSION THREAD

**Discussion by FJBDev** [mytourbook/mytourbook#1111] [Ideas]
At the moment, @wolfgang-ch sends an email titled _"MyTourbook XX.X Translation"_ to send each new version to the translators before releasing it (usually 2 weeks before each release)

I am thinking it could be useful to send it also to users that are interested to do _beta-testing_ before the official release so we could get their feedback. The feedback from those users would help detect bugs and maybe possible little improvements before releasing officially each new version.

I am also recalling some users like @stvzldn that were interested to beta-test MT but I can't find that conversation anymore, am I correct @stvzldn ?

@wolfgang-ch if you are interested in this idea, maybe the interested users could response to this thread to let you know they are interested.

**Comment by wolfgang-ch** [comment]
Good idea, I'll setup a mailing list if there are interests

**Reply by mff73** [reply]
Hello,
Good idea 💡
I am current using two MTB versions 😂
23.3. 2 and one test version you shared for suunto/fit exports.

**Reply by Thompson-ongithub** [reply]
I could test as well (but I'm not sure how much I could really add). A quick test after backing up the data directory isn't too much of a hazzle...

**Reply by stvzldn** [reply]
I am willing to do Beta Testing

**Reply by FJBDev** [reply]
> I could test as well (but I'm not sure how much I could really add). 

I think that anything helps. It's really good to have feedback from skilled users to make sure nothing big and bad gets released officially but also to make little modifications if needed. [Here ](https://github.com/mytourbook/mytourbook/issues/1124)is a recent example of feedback that could have been beneficial before releasing 23.5.0

**Reply by eborre** [reply]
I would love to be able to give back to the authors by helping with testing, I just don't know if I'm an active/extensive enough user to help.

But I'll gladly pitch in.

E

**Comment by wolfgang-ch** [comment]
This is the mailing list for MT betas: https://sourceforge.net/p/mytourbook/mailman/mytourbook-beta/
